default_settings.json

```diff
@@ -20,17 +20,32 @@
                 "denom": "uatom",
                 "displayName": "ATOM",
                 "fraction": 1000000
-            }, 
+            },
+            {
+                "denom": "ubedrock",
+                "displayName": "UBEDROCK",
+                "fraction": 1000000
+            },
             {
                 "denom": "umuon",
                 "displayName": "MUON",
                 "fraction": 1000000
             },
             {
-                "denom": "pylon",
-                "displayName": "PYLON",
+                "denom": "ustripeusd",
+                "displayName": "STRIPEUSD",
+                "fraction": 1000000
+            },
+            {
```

Comment:
Do we need ujuno here as well? Is it to aid your own local config?

Reply:
this is configurable we can edit as many as in live server, this is just a sample

Reply:
> Do we need ujuno here as well? Is it to aid your own local config?

this is configurable we can edit as many as in live server, this is just a sample